Update examples to bold terms consistently (#9)
globus/globus-flows-examples: transfer_examples/transfer_after_approval/README.adoc

*   `__Private_Parameters` and `_private` parameter prefixes are used throughout.
This helps restrict visibility of private information (such as email addresses and SMTP credentials)

> when users view the **flow definition** as well as while the **flow runs**.

Reviewer: I think this is more consistent:

```suggestion
when users view the **flow** definition as well as while the **flow** is executing.
```

We could still say "while the flow runs" but I think this terminology collision can be avoided.

Author: I had called out `flow definition` as a new term, to distinguish it from `flow`. But I'm not strongly attached to that. We can stick with `flow` as a term and "definition" as a characteristic of a `flow`. I just want us to be consistent in how we present it -- no more "Flow definition", "Flow Definition", etc. But before I change, I want to note that `**flow definition**` was very much intentional and fits the pattern we've established.
Should I update to `**flow** definition` in all cases? (I see you flagged some particulars, but it's more about the rule than any one case.)

---

I think you're right that `is executing` is a good way to improve the second case, but now that my eyes are on it, the thing which executes is a `**run**`, and I'm also generally trying to improve our consistency on that front. (We can easily confuse users by switching back and forth between talking about "what the flow does" and "what the run does" -- _we're_ clear on the distinction, but a user may not be.)

I would prefer that we make it `**run** is executing`, and then we need to retool the sentence slightly.

---

Combining the above two, how would this be?
```suggestion
when users view the **flow** definition as well as while their **runs** of the
**flow** are executing.
```

Reviewer: Yeah, that looks right to me.

Sounds like I overlooked `**flow definition**` instances, but my intention was to flag all of them. :+1:

Author: I'm not sure if you did -- but I wanted to make sure we're speaking the same language in terms of what the desired outcome is, not just the individual instances of `flow definition`. I'll do a check after I apply changes. :+1:

+
For more information, see link:https://docs.globus.org/api/flows/authoring-flows/secrets/[Protecting Secrets]
in the Authoring Flows documentation.

*   By setting the `RunAs` value in the `TransferFile` **state**,
the transfer operation will be performed using the **flow**'s own identity
to access the destination collection.
This allows the **flow** to be made public -- and run by any user --
without having to give individual users write permission on the destination **collection**.

== Prerequisites

> The **flow definition** must be modified before using it to create a new **flow**.

Reviewer: ```suggestion
The **flow** definition must be modified before using it to create a new **flow**.
```

It also has several requirements when running.

=== Modification requirements

*   The `SetupEmailLoop` **state** must be modified.
The email addresses of approvers must be updated, and unique IDs must be assigned to each reviewer.
Finally, the `loop_end` value must be updated to match the number of approvers.

*   The `SendEmail` **state** must be modified.
Specifically, the SMTP hostname, username, and password in `send_credentials` must be updated,
as well as the email address in `sender`.
+
It is also possible to use an AWS SES credential for sending emails.
See link:https://docs.globus.org/api/flows/hosted-action-providers/ap-notification-notify/[the Notification action provider documentation] for more details.

*   The `TransferFile` **state** must be modified.
The `destination_endpoint` must be set to the destination Guest Collection ID that will receive approved files,
and the `destination_path` should match the target directory you want approved files transferred to.

=== Execution requirements

> *   After modifying the **flow definition** and creating the flow in the Globus Flows service,

Reviewer: Perhaps?

```suggestion
*   After modifying the **flow** definition and creating the **flow** in the Globus Flows service,
```
